package.json

```diff
@@ -12,32 +12,34 @@
     "test": "test"
   },
   "dependencies": {
-    "babel-runtime": "^6.26.0",
-    "bn.js": "^5.1.1",
+    "bn.js": "^5.1.2",
     "create-hash": "^1.2.0",
     "decimal.js": "^10.2.0",
     "inherits": "^2.0.4",
     "lodash": "^4.17.15",
-    "ripple-address-codec": "^4.1.0"
+    "ripple-address-codec": "^4.1.1"
   },
   "devDependencies": {
-    "babel-cli": "^6.8.0",
-    "babel-core": "^6.8.0",
-    "babel-eslint": "^10.0.2",
-    "babel-loader": "^6.2.4",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-register": "^6.8.0",
-    "eslint": "^7.0.0",
-    "fs-extra": "^9.0.0",
-    "intercept-stdout": "^0.1.2",
-    "istanbul": "~0.4.3",
-    "mocha": "~7.1.2"
+    "@types/jest": "^26.0.0",
+    "@types/lodash": "^4.14.155",
+    "@types/node": "^14.0.10",
+    "@typescript-eslint/eslint-plugin": "^3.3.0",
+    "@typescript-eslint/parser": "^3.3.0",
+    "eslint": "^7.2.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-plugin-import": "^2.21.2",
+    "eslint-plugin-jest": "^23.13.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
+    "jest": "^26.0.1",
+    "typescript": "^3.9.5"
   },
   "scripts": {
-    "compile": "babel --optional runtime -d distrib/npm/ src/ && cp src/enums/definitions.json distrib/npm/enums",
-    "prepublish": "npm test && npm run lint && npm run compile",
-    "test": "istanbul test _mocha",
-    "lint": "if ! [ -f eslintrc ]; then curl -o eslintrc 'https://raw.githubusercontent.com/ripple/javascript-style-guide/es6/eslintrc'; echo 'parser: babel-eslint' >> eslintrc; fi; eslint -c eslintrc src/**/*.js test/*.js"
+    "compile": "tsc && cp ./src/enums/definitions.json ./dist/enums",
+    "prepare": "npm run compile && npm test",
+    "test": "jest",
+    "lint": "eslint ./src/**/*.ts && eslint ./test/*.js"
```

Comment:
Note : Linting does not work correctly on this PR, and will be added in next PR to reduce size.